print/server-middleware/routes/pdfGeneratorNativeChrome.js

```diff
@@ -70,6 +70,12 @@ router.use('/pdfChrome', async (req, res) => {
     ]
     await page.setCookie(...cookies)
 
+    const extraHeaders = {}
+    if (process.env.BASIC_AUTH_TOKEN) {
+      extraHeaders['Authorization'] = `Basic ${process.env.BASIC_AUTH_TOKEN}`
+    }
+    await page.setExtraHTTPHeaders(extraHeaders)
```

Comment:
No, but it is a promise and i want to be sure that the header is set when we send there request for the page.
Even if it accidentally works, i would prefer to use the interface correctly.

Reply:
Could we then move this line inside the if-scope? Otherwise we always add more time for the print functionality for a feature that is only used in staging.

Reply:
done